nginx-nodejs-redis/README.md

```diff
@@ -0,0 +1,97 @@
+## Compose sample application
+
+## Node.js application with Nginx proxy and Redis database
+
+Project structure:
+```
+.
+├── README.md
+├── docker-compose.yml
+├── nginx
+│   ├── Dockerfile
+│   └── nginx.conf
+└── web
+    ├── Dockerfile
+    ├── package.json
+    └── server.js
+
+2 directories, 7 files
+
+
+```
+[_docker-compose.yml_](docker-compose.yml)
+```
+redis:
+    image: 'redislabs/redismod'
+    ports:
+      - '6379:6379'
+  web1:
+    restart: on-failure
+    build: ./web
+    hostname: web1
+    ports:
+      - '81:5000'
+  web2:
+    restart: on-failure
+    build: ./web
+    hostname: web2
+    ports:
+      - '82:5000'
+  nginx:
+    build: ./nginx
+    ports:
+    - '80:80'
+    depends_on:
+    - web1
+    - web2
+```
+The compose file defines an application with four services `redis`, `nginx`, `web1` and `web2`.
+When deploying the application, docker-compose maps port 80 of the nginx service container to port 80 of the host as specified in the file.
+
+
+> ℹ️ **_INFO_**  
+> Redis runs on port 6379 by default. Make sure port 6379 on the host is not being used by another container, otherwise the port should be changed.
+
+## Deploy with docker-compose
+
+```
+$ docker-compose up -d
```

Comment:
My previous suggestion was to use the `docker compose up -d` output and not the one from `docker compose up`, why did you completely removed logs?

You can commit the following suggestion which corresponds to what we need in the documentation


```suggestion
$ docker-compose up -d
[+] Running 24/24
 ⠿ redis Pulled                                                                                                                                                                                                                      ...
   ⠿ 565225d89260 Pull complete                                                                                                                                                                                                      
[+] Building 2.4s (22/25)
 => [nginx-nodejs-redis_nginx internal] load build definition from Dockerfile                                                                                                                                                         ...
[+] Running 5/5
 ⠿ Network nginx-nodejs-redis_default    Created                                                                                                                                                                                      
 ⠿ Container nginx-nodejs-redis-web2-1   Started                                                                                                                                                                                      
 ⠿ Container nginx-nodejs-redis-redis-1  Started                                                                                                                                                                                      
 ⠿ Container nginx-nodejs-redis-web1-1   Started                                                                                                                                                                                      
 ⠿ Container nginx-nodejs-redis-nginx-1  Started
```